### PR TITLE
Consensus release for Node 8.10

### PR DESCRIPTION
**Pull request by dnadales**
This PR also closes https://github.com/IntersectMBO/ouroboros-consensus/issues/950

### CODE REVIEW

File: ouroboros-consensus-cardano/src/unstable-byronspec/Ouroboros/Consensus/ByronSpec/Ledger/Ledger.hs [hunk 1]
```diff
@@ -53,7 +54,7 @@ data instance LedgerState ByronSpecBlock = ByronSpecLedgerState {
   deriving NoThunks via AllowThunk (LedgerState ByronSpecBlock)
 
 newtype ByronSpecLedgerError = ByronSpecLedgerError {
-      unByronSpecLedgerError :: [Spec.PredicateFailure Spec.CHAIN]
+      unByronSpecLedgerError :: (NonEmpty (Spec.PredicateFailure Spec.CHAIN))
```
Comment (jasagredo) [inline ouroboros-consensus-cardano/src/unstable-byronspec/Ouroboros/Consensus/ByronSpec/Ledger/Ledger.hs:57]:
```suggestion
      unByronSpecLedgerError :: NonEmpty (Spec.PredicateFailure Spec.CHAIN)
```

**Review by jasagredo — APPROVED** [review]

File: ouroboros-consensus-diffusion/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,36 @@
 
 # Changelog entries
 
+<a id='changelog-0.11.0.0'></a>
+## 0.13.0.0 — 2024-04-03
```
Comment (jasagredo) [inline ouroboros-consensus-diffusion/CHANGELOG.md:6]: Ah you didn't modify the HTML tag 😄

File: ouroboros-consensus-cardano/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,51 @@
...
+
+- Use the version of cardano-git-rev in the cardano-base repo.
+
+- Bump `NodeToNodeVersion` part of `latestReleasedNodeVersion` to
+  `NodeToNodeV_13` from `NodeToNodeV_11`.
+
+- Add `ConvertRawTxId` instances for Shelley and Byron.
+
+- Update ledger packages
+  - -ledger-allegra`: `^>=1.4`
```
Comment (jasagredo) [inline ouroboros-consensus-cardano/CHANGELOG.md:29]: This was accidentally truncated it seems

File: ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs [hunk 4]
```diff
@@ -1143,3 +1118,16 @@ mkPartialLedgerConfigShelley transitionConfig maxMajorProtVer shelleyTriggerHard
...
+  , blk ~ ShelleyBlock (BlockProtocol blk) (ShelleyBlockLedgerEra blk)
+  ) => IsShelleyBlock blk
+instance ShelleyBasedEra era => IsShelleyBlock (ShelleyBlock proto era)
+
+type family ShelleyBlockLedgerEra blk where
+  ShelleyBlockLedgerEra (ShelleyBlock proto era) = era
+
+-- | We need this wrapper to partially apply a 'TransitionConfig' in an NP.
+newtype WrapTransitionConfig blk =
+    WrapTransitionConfig (L.TransitionConfig (ShelleyBlockLedgerEra blk))
```
[the remark this answers is not in the document]
Comment (dnadales) [reply ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs:1133]: Yes, I wanted to incorporate to this PR, I thought it'd be better to do it after this PR.

**Review by jasagredo — APPROVED** [review]

**Review by amesgen — APPROVED** [review]

**Pull request merged**